Tests/Python/PandasConverterTests.cs

```diff
@@ -2652,6 +2712,11 @@ public void BackwardsCompatibilitySeries_set_value(string index, bool cache = fa
         {
             if (cache) SymbolCache.Set("SPY", Symbols.SPY);
 
+            // set_value deprecated in newer pandas
+            if(_newerPandas) {
+                return;
+            }
+
             using (Py.GIL())
```

Comment:
Cool! Makes me think we should be adding new unit tests for the new stuff in pandas?

Reply:
I went through the unit tests that use deprecated functions and ensured their replacements are all covered. All these cases were just using old deprecated ways of indexing instead of stricter iloc, loc, iat, & at indexing. These are already being tested 👍🏽

There were a couple tests that were made to test other functions but used the deprecated indexing, so I refactored them to use the stricter indexers to be sure we are testing the method intended for both new and old pandas. 😁  They are all good now!